OwnTube.tv/types.ts

```diff
@@ -0,0 +1,19 @@
+export interface Video {
+  readonly id: number;
+  readonly name: string;
+  readonly category: { id: number | null; label: string };
+  readonly thumbnailPath: string;
+  readonly thumbnailUrl?: string;
+  readonly description?: string | null;
+}
+
+export interface Category {
+  readonly label: string;
+  readonly id: number;
+}
```

Comment:
I'm a bit unsure about specifying things as `readonly` here.. Let's talk about it during stand-up today.